portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+google.charts.load('current', {'packages':['corechart']});
```

Comment:
You can put this in a separate js file if you want to. Just an option

Reply:
Done